src/site/articles/mixins/index.markdown

```diff
@@ -248,30 +234,75 @@ in the underlying language.
 ### Statics
 
 Can one use the statics of the original class
-via the mixin application or not?  
+via the mixin application or not?
+
 Again, the answer (No) follows from the semantics of inheritance.
-Statics are not inherited in Dart. 
+Statics are not inherited in Dart.
 
 ### Types
 
-What is the type of a mixin application instance?
-In general, it is a subtype of its superclass,
-and also supports the methods defined on the mixin.
-The mixin name itself, however,
-denotes the type of the original class,
-which has its own superclass
-and may not be compatible with a particular mixin application.
-
-What about the interfaces a class supports?
-Does its mixin support them?
-In general, no, since interface support
-may rely on inherited functionality.
-This implies that a mixin application must declare
-what interfaces it implements explicitly.
+What is the type of a mixin application instance? In general, it is a
+subtype of its superclass, and also a subtype of the type denoted by
+the mixin name itself, i.e., the type of the original class.
```

Comment:
We usually avoid latin abbreviations (and i.e. in particular) because many people misunderstand them.